pysat/tests/test_instruments.py

```diff
@@ -77,7 +77,10 @@ def test_inst_start_time(self, inst_dict, kwarg, output):
 
         self.test_inst.load(date=date)
 
-        assert self.test_inst['uts'][0] == output
+        if self.test_inst.pandas_format:
+            assert self.test_inst['uts'].iloc[0] == output
+        else:
+            assert self.test_inst['uts'][0] == output
```

Comment:
Would `assert self.test_inst[0, 'uts'] == output` work for this?

Reply:
fixed